test/test_quantized_deconvolutional_layer.h

```diff
@@ -260,12 +263,14 @@ TEST(quantized_deconvolutional, read_write2) {
     };
 #undef O
 #undef X
-    quantized_deconvolutional_layer<tan_h> layer1(14, 14, 5, 3, 6, connection_table(connection, 3, 6));
-    quantized_deconvolutional_layer<tan_h> layer2(14, 14, 5, 3, 6, connection_table(connection, 3, 6));
+    quantized_deconvolutional_layer<tan_h> layer1(14, 14, 5, 3, 6,
+connection_table(connection, 3, 6));
+    quantized_deconvolutional_layer<tan_h> layer2(14, 14, 5, 3, 6,
+connection_table(connection, 3, 6));
```

Comment:
I see. I didn't notice I was reading comments.

Talking about file header comments (Copyright...).
I guess they will be replaced by shorter comments.

```
// Copyright (c) 2013-2016, Taiga Nomi. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
```